### ISSUE TITLE
Optional import filter to view only files with certain extensions

### ISSUE THREAD

**Issue by TillS** [matthiasbock/BooleSim#27]
Example as done here:

https://www.filepicker.io/products/javascript_v1/#localfiles

Important source code to consider for .png files for instance: 

Choose a PNG file to display below: 

 &lt;input type="file" onchange="filepicker.read(this, {base64encode: true}, function(imgdata) {$('#readResultImg').attr('src', 'data:image/png;base64,'+imgdata);}, function(fperror){alert(fperror);});" accept="image/png"></input>

**Comment by chaitan3** [comment]
The extension of booleannet files vary a lot. There is no fixed standard for python booleannet or R boolnet files, so I don't think this would be of much help

**Comment by TillS** [comment]
Well. it's either .txt or .py or .R files so not much variation I guess, but it's true, that if the effort to implement this is too big, it's not really worth a shot, because it doesn't improve the tool a lot. However, it was a minor revision point by the reviewer and I don't want him to reject our paper again because of such a minor thing.

**Comment by chaitan3** [comment]
I don't think the reviewer will reject our paper on this minor point. Moreover, we do have a file type detector in our code which sort of makes the requested feature redundant.

Also, I think we have covered most of the changes asked by the reviewer. If there are no more issues, we can resubmit.

**Comment by matthiasbock** [comment]
I agree. As there is no "standard", a wide variety of possible file extensions exists. Additionally to the above also .ginml .jsbgn .bool .net .boolnet .booleannet etc. etc. It can also be no file extension at all. Therefore it doesn't really make sense to add a filter. I think it will be more frustrating for a user not to find his model file, than having to pick it from a huge list of files. In my opinion the reviewer will (have to) understand that, if we answear like this.
